Sources/XiEditor/Client.swift

```diff
@@ -20,14 +20,14 @@ protocol XiClient: AnyObject {
     /// An update to the contents of a given view. The structure of the `update`
     /// param is described in detail in the
     // [documentation](http://xi-editor.github.io/xi-editor/docs/frontend-protocol.html#view-update-protocol).
-    func update(viewIdentifier: String, update: [String: AnyObject], rev: UInt64?)
```

Comment:
`Client` is now clean from `[String: AnyObject]` parameters. The idea would be to keep it that way (all parsing done before it hits this layer).